app/models/manageiq/providers/autosde/storage_manager/host_initiator.rb

```diff
@@ -0,0 +1,21 @@
+class ManageIQ::Providers::Autosde::StorageManager::HostInitiator < ::HostInitiator
+  supports :create
+
+  def self.raw_create_host_initiator(_ext_management_system, _options = {})
+    host_initiator_to_create = _ext_management_system.autosde_client.StorageHostCreate(
+      :name           => _options['name'],
```

Comment:
If you're going to use `_options` then it shouldn't have a leading `_` (that indicates that the parameter isn't going to be used)

Reply:
fixed